components/Core/Button/button.module.css

```diff
@@ -0,0 +1,18 @@
+.button {
+  background-color: white;
+  border: solid 1px;
+  color: black;
+  padding: 6px 12px;
+  border-radius: 0.5rem;
```

Comment:
It's often a good practice to use pixels (or percentage) for radius, and leave REM usage for font-size. Now if the html {font-size: 16px} is to change, it will affect your radii